docker/llm-image-captioning/README.md

```diff
@@ -0,0 +1,21 @@
+## Dataset:
+This agent works on datasets comprised exclusively of images.
+
+## Workflow: 
+An example includes: https://app.encord.com/workflow-templates/view/a7b7d551-7b29-429b-8e7d-08d5d2295109
+
+This agent will work on any workflow that has an agent stage and will caption the images at that stage and move them onto the next stage.
+
+The agent is configured to require exclusively images in the project and will throw a pre-execution error if this is not the case.
+
+## Ontology
+
+The agent works with an Ontology that contains a top level text classification object. It will throw a pre-execution error if this is not the case
+
+## Execution
+
+docker run -e ENCORD_SSH_KEY -e OPENAI_API_KEY encord-agents-llm-image-captioning --project-hash=your-project-hash
```

Comment:
Let's tell them to first export those env variables (or set them directly).
Also, could we please put this within backticks.